RELEASING.md

```diff
@@ -1,8 +1,22 @@
 # Felix release artifacts
 
-The artifacts of a Felix release are a tarball (aka PyInstaller
-bundle) and a Docker container image, both containing the Felix
-executable(s) .
+Running the main `make release` target, described below, guides you
+through creating and distributing the following artifacts:
+
+- A git tag for the release, annotated with the release note; ready to 
+  push to Github.
+- File `dist/calico-felix-$VERSION.tgz`; the "PyInstaller bundle" 
+  containing the Felix binaries.  We attach that file to the 
+  Github release.
+- Docker container images: `calico/felix:$VERSION` and 
+  `quay.io/calico/felix:$VERSION` containing the Felix binaries.  These
+  are ready to push to Dockerhub and Quay.  They also form the input
```

Comment:
I understood from what you said yesterday that their _only_ purpose was as input to the downstream process.  If that's right, I'd prefer that to be more clearly stated here.  Or if they have other purposes, what are those?

(For example, if they have no other purpose, why should we not consider simplifying our processes by using the bundle as input for calico/node instead?  My point is that we should do that right now - but that someone might consider that in future, and so would benefit from as full a description as possible of the artifacts' purposes.)